### PR TITLE
amount-of-coins-sent-in-a-callsc-displayed-is-wrong

### PR DESCRIPTION
**Pull request by B-Naoufal**

### CODE REVIEW

**Review by Thykof — APPROVED** [review]

File: wails-frontend/src/pages/PasswordPromptHandler/SignSC/CallSc.tsx [hunk 1]
```diff
@@ -22,10 +22,13 @@ export function CallSc(props: PromptRequestData) {
     Description,
   } = props;
 
-  const toAddInHeigthDescription = Description ? 200 : 0;
+  // Convert nano mass to mass
+  const coinsInMass = coinsInNanoMass / 1000000000;
```
Comment (Thykof) [inline wails-frontend/src/pages/PasswordPromptHandler/SignSC/CallSc.tsx:26]: can you see if it's possible to use the massa-web3 convert function?

Comment (Thykof) [reply wails-frontend/src/pages/PasswordPromptHandler/SignSC/CallSc.tsx:26]: https://github.com/massalabs/massa-web3/blob/815357a17b0b44be8759a4abcc643291f7e57c4f/src/utils/converters.ts#L50

**Review by peterjah — CHANGES_REQUESTED** [review]
This whole thing must be refactored using string/bigint instead of number for every "amounts"

**Comment by B-Naoufal** [comment]
will be covered here : https://github.com/massalabs/station-massa-wallet/pull/716

**Pull request closed without merging**